src/sql/connection.py

```diff
@@ -348,3 +353,17 @@ def _get_curr_connection_info(self):
             "driver": getattr(engine.dialect, "driver", None),
             "server_version_info": getattr(engine.dialect, "server_version_info", None),
         }
+
+    def _transiple_query(self, query):
+        connection_info = self._get_curr_connection_info()
+        try:
+            write_dialect = connection_info["dialect"]
+            if write_dialect in DIALECT_NAME_SQLALCHEMY_TO_SQLGLOT_MAPPING:
+                write_dialect = DIALECT_NAME_SQLALCHEMY_TO_SQLGLOT_MAPPING[
+                    write_dialect
+                ]
+            query = sqlglot.transpile(query, read="duckdb", write=write_dialect)[0]
+        except Exception as e:
+            raise e
```

Comment:
```suggestion
```

Reply:
thx, now it's fix, it's cleaner, good catch!